app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def options_for_ratings(selected=nil)
+    options_for_select([5, 4, 3, 2, 1].map {|option| [pluralize(option, "Star"), option]}, selected)
```

Comment:
This line http://cl.ly/image/3p0T3a1g2g1v is over 80 characters - to keep lines readable and to make debugging easier please refactor any lines over this limit. You can see this referenced in the style guide here - https://github.com/bbatsov/ruby-style-guide#80-character-limits